### PR TITLE
Add NuGet 6.12 release notes

### PR DESCRIPTION
**Pull request by zivkan**

### CODE REVIEW

File: docs/release-notes/NuGet-6.12.md [hunk 1]
```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
...
+
+### Breaking changes
+
+* NuGetAudit should check transitive packages by default when the .NET 9 SDK is installed [#13293](https://github.com/NuGet/Home/issues/13293)
+
+* Change NuGetAuditMode default to all for non-SDK style projects - [#13584](https://github.com/NuGet/Home/issues/13584)
+
+* [Epic]Deprecate http usage: Promote from warning to error - [#13289](https://github.com/NuGet/Home/issues/13289)
+
+### Issues fixed in this release
```
[the remark this answers is not in the document]
Comment (kartheekp-ms) [reply docs/release-notes/NuGet-6.12.md:56]: Thanks for identifying that the release notes are missing a customer facing change. Sorry for the delay in creating the issue. https://github.com/NuGet/Home/issues/13891

File: docs/release-notes/NuGet-6.12.md [hunk 2]
```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
...
+
+* tweak wording of NU1603 - [#13446](https://github.com/NuGet/Home/issues/13446)
+
+* Default Package icon shown even when embedded icon file exists on disk - [#13766](https://github.com/NuGet/Home/issues/13766)
+
+* Navigation telemetry for hyperlinks: License, ReportAbuse, Readme, ProjectUrl - [#13749](https://github.com/NuGet/Home/issues/13749)
+
+* Navigation telemetry for Owner Profile URLs in PM UI - [#13738](https://github.com/NuGet/Home/issues/13738)
+
+* NuGet's dependency graph resolution algorithm does not scale well for large graphs - [#13692](https://github.com/NuGet/Home/issues/13692)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.12.md:70]: This should probably in the What's new as one of the 2-3 flagship changes.

File: docs/release-notes/NuGet-6.12.md [hunk 3]
```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
...
+
+NuGet 6.12.0 is available in the .NET 9.0.100 SDK.
+
+* Add property for toggling the to the previous NuGet resolver: RestoreUseLegacyDependencyResolver - [#13700](https://github.com/NuGet/Home/issues/13700)
+
+* Owner profile hyperlinks needed in Details Pane of PM UI - [#13686](https://github.com/NuGet/Home/issues/13686)
+
+* Bubble-up Known Vulnerability Indicators in Solution Explorer for Transitive Packages - [#13636](https://github.com/NuGet/Home/issues/13636)
+
+* Map branch name from sourcelink to RepositoryBranch for NuGet pack - [#13625](https://github.com/NuGet/Home/issues/13625)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.12.md:36]: This probably can be moved to changes, not sure if it's worth calling out in the big features.

File: docs/release-notes/NuGet-6.12.md [hunk 4]
```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
...
+
+### Issues fixed in this release
+
+* Deserializing an empty version range in a package dependency fails in .NET SDK 9.0.100-rc.2 [#13869](https://github.com/NuGet/Home/issues/13869)
+
+## Summary: What's New in 6.12
+
+NuGet 6.12.0 is available in the .NET 9.0.100 SDK.
+
+* Add property for toggling the to the previous NuGet resolver: RestoreUseLegacyDependencyResolver - [#13700](https://github.com/NuGet/Home/issues/13700)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.12.md:30]: If we have the new dependency resolver chnage near the top here, do we still need this?

File: docs/release-notes/NuGet-6.12.md [hunk 5]
```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
...
+
+* Roll-out new breaking change process for SDK tools, respect SdkAnalysisLevel - [#13309](https://github.com/NuGet/Home/issues/13309)
+
+### Breaking changes
+
+* NuGetAudit should check transitive packages by default when the .NET 9 SDK is installed [#13293](https://github.com/NuGet/Home/issues/13293)
+
+* Change NuGetAuditMode default to all for non-SDK style projects - [#13584](https://github.com/NuGet/Home/issues/13584)
+
+* [Epic]Deprecate http usage: Promote from warning to error - [#13289](https://github.com/NuGet/Home/issues/13289)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.12.md:54]:
```suggestion
* Deprecate http usage: Promote from warning to error - [#13289](https://github.com/NuGet/Home/issues/13289)
```

Should we just move this to the What's New?

I think it's really up to us where we want to include it.

**Review by nkolev92 — COMMENTED** [review]
Looks good to me.

File: docs/release-notes/NuGet-6.12.md [hunk 6]
```diff
@@ -17,6 +15,139 @@ NuGet distribution vehicles:
...
-## Not yet released
+## Summary: What's New in 6.11.1
+
+NuGet 6.12.1 is available in Visual Studio 17.12.0 and the.NET 9.0.101 SDK.
+
+### Issues fixed in this release
+
+* Deserializing an empty version range in a package dependency fails in .NET SDK 9.0.100-rc.2 [#13869](https://github.com/NuGet/Home/issues/13869)
+
+## Summary: What's New in 6.12
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.12.md:26]: 6.12 has an unlikely bug in the new resolver, https://github.com/NuGet/Home/issues/13890. 

Should we include it in these docs?
How should we include it there?

Comment (zivkan) [reply docs/release-notes/NuGet-6.12.md:32]: I added a new section for known issues

**Review by jeffkl — APPROVED** [review]

**Review by donnie-msft — APPROVED** [review]

**Review by jebriede — APPROVED** [review]

File: docs/release-notes/NuGet-6.12.md [hunk 7]
```diff
@@ -17,6 +15,147 @@ NuGet distribution vehicles:
...
+
+NuGet 6.12.0 is available in the .NET 9.0.100 SDK.
+
+* NuGet's dependency graph resolution algorithm does not scale well for large graphs - [#13692](https://github.com/NuGet/Home/issues/13692)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.12.md:36]: Not sure if I commented on this originally, but It's probably ok to take creative freedom and write it in a way that's nicer. 

```suggestion
* Introduce optimized dependency resolver that scales well for large graphs - [#13692](https://github.com/NuGet/Home/issues/13692)
* ```

Comment (zivkan) [reply docs/release-notes/NuGet-6.12.md:36]: > take creative freedom

ok!

```suggestion
* The hypotenuse of the monad is tangential to the flux capacitor second derivitive  - [#13692](https://github.com/NuGet/Home/issues/13692)
```

**Pull request merged**